internal/sink/sinkSender.go

```diff
@@ -420,7 +418,6 @@ Loop:
 			}
 			s.queueDepthGauge.Add(-1.0)
 			s.mutex.RLock()
-			urls = s.matcher.getUrls()
```

Comment:
urls is now added to the sink structs and we will be referencing them directly from the struck rather than sending them to the send function